setup_scripts/install_dependencies.sh

```diff
@@ -59,6 +59,10 @@ sudo ./setup_realsense_manual.sh
 
 cd $CURR_DIR
 sudo ./setup_phidgets.sh
+
+cd $CURR_DIR
+sudo ./setuo_alglib.sh
```

Comment:
Should this be setup rather than setuo?

Reply:
yes, I fixed the typo in the last commit. Also renamed it install_alglib.sh